tests/tests.go

```diff
@@ -28,14 +29,16 @@ import (
 const (
 	TestNamespace          = "kubemacpool-test"
 	OtherTestNamespace     = "kubemacpool-test-alternative"
-	ManagerNamespce        = names.MANAGER_NAMESPACE
 	nadPostUrl             = "/apis/k8s.cni.cncf.io/v1/namespaces/%s/network-attachment-definitions/%s"
 	linuxBridgeConfCRD     = `{"apiVersion":"k8s.cni.cncf.io/v1","kind":"NetworkAttachmentDefinition","metadata":{"name":"%s","namespace":"%s"},"spec":{"config":"{ \"cniVersion\": \"0.3.1\", \"type\": \"bridge\", \"bridge\": \"br1\"}"}}`
 	podNamespaceOptInLabel = "mutatepods.kubemacpool.io"
 	vmNamespaceOptInLabel  = "mutatevirtualmachines.kubemacpool.io"
 )
 
 var (
+	// The MANAGER_NAMESPACE variables is passed to the test suite in case
+	// different kmp manager is at not default namespace
+	managerNamespace         = getEnv("MANAGER_NAMESPACE", names.MANAGER_NAMESPACE)
```

Comment:
Since this function is falling back to a default value how about we change its signature to 
`tryGetEnvVar(target, default string)`?